src/state/ducks/room.js

```diff
@@ -53,29 +54,46 @@ const logout = () => {
   };
 };
 
+const toggleThumbnailMode = () => {
+  return function(dispatch, getState) {
+    let { thumbnailMode } = getState().room;
+
+    thumbnailMode ? janusApi.disableThumbnailMode() : janusApi.enableThumbnailMode();
```

Comment:
My first thought was to use a `toggleThumbnailMode` function just to be consistent with `toggleVideo` and `toggleAudio`. However, the thumbnail mode is something slightly different (I do not remember if janusApi keeps the current status of the feature). I will check the code.

Reply:
It was also my first approach, but I ended doing it different because we want to enable or disable de video subscription no matter the current status of them. So, I decided to avoid asking to janusApi. 

We can arrange time to talk about it in a video call / pair programming session if you want.